samza-core/src/main/scala/org/apache/samza/storage/TaskStorageManager.scala

```diff
@@ -249,7 +88,7 @@ class TaskStorageManager(
         val newestOffset = if (sspMetadata == null) null else sspMetadata.getNewestOffset
         debug("Got offset %s for store %s" format(newestOffset, storeName))
 
-        val loggedStorePartitionDir = TaskStorageManager.getStorePartitionDir(loggedStoreBaseDir, storeName, taskName)
+        val loggedStorePartitionDir = StorageManagerUtil.getStorePartitionDir(loggedStoreBaseDir, storeName, taskName)
```

Comment:
Now that we've split the restore and the flush sequences across these 2 classes, we could go one step further and fold the offset-flush as a part of loggedStore.flush(); maybe track it in a separate clean-up JIRA?

Reply:
done https://issues.apache.org/jira/browse/SAMZA-2050